cpu/src/stark.rs

```diff
@@ -119,7 +119,7 @@ impl CpuChip {
             )
             .assert_one(local.read_1_used());
         builder
-            .when(is_jal + is_left_imm_op)
+            .when(is_jal + is_left_imm_op + is_loadfp + is_imm32)
```

Comment:
Here and below we add some missing cases to the memory channel constraints, asserting that the read channels are unused for `IMM32` and `LOADFP` and that the write channel is unused for `BEQ` and `BNE`.